examples/MvcWeb/TheraLang/Controllers/ParticipationController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using MvcWeb.TheraLang.Entities;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using MvcWeb.TheraLang.Services;
+using System.Collections.Generic;
+
+namespace MvcWeb.TheraLang.Controllers
+{
+    [Route("api/participation")]
+    [ApiController]
+    public class ParticipationController : ControllerBase
+    {
+        public ParticipationController(IProjectParticipationService _service)
+        {
+            this.service = _service;
+        }
+
+        private readonly IProjectParticipationService service;
+
+        [HttpGet]
+        [Route("get")]
+        public IActionResult Get()
+        {
+            IEnumerable<ProjectParticipation> members = service.GetAll();
+
+            return Ok(members);
+        }
+
+        [HttpPost]
+        [Route("create/{userId}/{projectId}")]
+        public async Task<IActionResult> Post([FromBody] int userId, int projectId)
+        {
```

Comment:
if (userId == default)
            {
                throw new ArgumentException($"{nameof(userId)} cannot be 0");
            }